modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -23,21 +17,21 @@ endif::[]
 
 ifndef::tf-classic,tf-hcp[]
 You can quickly create a
-ifdef::rosa-hcp[]
-{hcp-title}
-endif::rosa-hcp[]
-ifndef::rosa-hcp[]
-{product-title} (ROSA)
-endif::rosa-hcp[]
-cluster with the {sts-first} by using the default installation options. The following summary describes the default cluster specifications.
+ifdef::openshift-rosa-hcp[]
+{hcp-title} cluster by using the default installation options.
```

Comment:
Should we be using `{hcp-title}` here? I would think it should just be `{product-title}`.